py/torch_tensorrt/dynamo/conversion/impl/unary/base.py

```diff
@@ -33,10 +53,13 @@ def convert_unary(
         The output of TensorRT Unary layer.
     """
     if not isinstance(input_val, TRTTensor):
-        raise RuntimeError(
-            f"{operation_type} received input {input_val} that is not part "
-            "of the TensorRT region!"
-        )
+        if get_python_op_from_trt_unary_op(operation_type):
+            SUPPORTED_ATEN_TORCH_UNARY_OPS[operation_type](input_val)
```

Comment:
Consider adding a debug log message here stating that the unary op was interecepted and replaced with a Python variant